content/python/concepts/strings/terms/join/join.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: '.join()'
+Description: 'Concatenates all items from an iterable into a single string.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+  - 'Data Science'
```

Comment:
Should we could part with one of these and keep two for the `Subjects`?

Reply:
@Dusch4593  Which one do you want to remove? I'm fine with any.

Reply:
@Dusch4593 were you thinking of parting with 'Code Foundations'?

Reply:
I was actually thinking maybe "Computer Science" since Python is more data science-y (right?) and strings are sort of related to code foundations(?)

Reply:
So I took a quick look, currently, the subjects listed for [Python Strings](https://www.codecademy.com/resources/docs/python/strings) are 'Code Foundations', 'Computer Science', 'Data Science'. 

Every string term entry, however, have only  'Data Science' & 'Computer Science' listed

Reply:
Ah, well if that's the case then yeah we should do away with "Code Foundations". 😄